core/src/main/java/org/apache/accumulo/core/cli/ConfigOpts.java

```diff
@@ -45,6 +47,14 @@ public synchronized String getPropertiesPath() {
     return propsPath;
   }
 
+  public static class DeprecatedOption implements IParameterValidator {
+    @Override
+    public void validate(String option, String value) throws ParameterException {
+      System.out.println(
```

Comment:
Should this use `System.err` instead? Or better yet, perhaps `log.warn`, so the behavior is user-configurable within the user's logging settings?

I'd hate for the printing of this to interfere with any automation/scripting a user might be doing, or anything else that might be capturing STDOUT.

Reply:
Understand but note that the GC does the same although I guess if using safemode then stdout is expected.  There could be future deprecated options that do not involve any STDOUT.   The --help will print the deprecation info to the console so if the option is actually used the warning could just be logged, log.warning.  I"m good with that.
```
 if (safemode) {
        System.out.println("SAFEMODE: There are " + confirmedDeletes.size()
            + " data file candidates marked for deletion.%n"
            + "          Examine the log files to identify them.%n");
```

Reply:
Since the help info already has the deprecation info, logging a warning when used here is fine.

If I remember correctly, I did argue against using STDOUT at the time it was added to the gc safemode, but it was make clear to me that safemode is typically run in a manual way, not scripted with a user's typical launch scripts, so STDOUT has specific utility for the user running it in safemode (though it could also be STDERR in that case, too). In general, we should use less `System.{out,err,exit,<probably others>}`, but there's always going to be exceptions to best practices.